pkg/cmd/bootstrap.go

```diff
@@ -138,15 +139,23 @@ func nonInteractiveMode(io *BootstrapParameters, client *utility.Client) error {
 }
 
 func setSecretIfNotSet(repoURL, accessToken string) error {
-	secret, _ := keyring.Get("kam", repoURL)
-	if secret != accessToken {
-		err := keyring.Set("kam", repoURL, accessToken)
+	hostName, err := webhook.HostFromURL(repoURL)
+	if err != nil {
+		return err
+	}
+	secret, err := keyring.Get("kam", hostName)
+	if err != nil && err != keyring.ErrNotFound {
+		return err
+	}
+	if accessToken != secret {
+		err := keyring.Set("kam", hostName, accessToken)
 		if err != nil {
-			return fmt.Errorf("Unable to link access Token to keyring for repo:%v", repoURL)
+			return fmt.Errorf("Unable to set access Token to keyring for repo: %q", repoURL)
```

Comment:
This doesn't indicate the error?

Also, how about fmt.Errorf("unable to set access token for repo %q using keyring: %w", repoURL, err)